lib/html2rss/config.rb

```diff
@@ -131,6 +131,20 @@ def category_selectors
       categories
     end
 
+    ##
+    # @return Array<Symbol>
+    def guid_selectors
+      guids = feed_config.dig(:selectors, :guid)
```

Comment:
- [ ] extract a method to extract selector names
- [ ] make it support a default value
- [ ] use it for guid selectors
- [ ] use it for category selectors